stdlib/http/src/test/java/org/ballerinalang/stdlib/websocket/WebSocketCompilationTest.java

```diff
@@ -57,12 +57,6 @@ public void testFailOnOpenOnIdle() {
         CompileResult compileResult = BCompileUtil.compileOnly(TEST_PATH + "fail_onOpen_onIdle.bal");
 
         assertExpectedDiagnosticsLength(compileResult, 5);
-        BAssertUtil.validateError(compileResult, 1, "Invalid resource signature for onOpen resource in service " +
```

Comment:
Shouldn't the error messages be validated? Given that there are 5 errors?

Reply:
Because it hits both the http and websocket compiler plugins, the order of the messages differ on each run. The reason why this test failed.